### DISCUSSION TITLE
Returning incorrect gridpoints for GPS coordinates

### DISCUSSION THREAD

**Discussion by boofuss60** [weather-gov/api#611] [General]
I don't understand this known issue:

_The API is returning gridpoints that are off by +1,+1. This causes forecasts for AFC (AER/ALU), AFG, AJK, BOX, CAE, DLH, FSD, HGX, HNX, LIX, LWX, MAF, MFR, MLB, MRX, MTR, and PIH to fall outside the bounding box of the grid. For these offices the exact grid returned from the points endpoint should be (gridX - 1, gridY - 1)._

I am not much of a coder but am delighted to find this API. I'm trying to use it (and trust it). I have GPS coordinates but I notice they are not always located in the returned gridpoints. But if I understand this issue correctly, I would change **https://api.weather.gov/gridpoints/LWX/70,86** to **https://api.weather.gov/gridpoints/LWX/69,85** to get the proper gridpoint. Correct?

I think I am missing something, because if it is that simple I wonder why I am doing this in my code instead of the developers fixing it. And, if I encode this correction, how would I know when it gets "fixed." I think I'm not understanding something.

**Comment by scadergit** [comment]
We are planning to fix it, the manual effort was only until it can be resolved. The official way to be notified is requeste to be included on the NCO ticket using the NCO support email.

**Reply by boofuss60** [reply]
Great! Actually, I'm using this to help me when backpacking in the mountains--which won't be for a couple months. Do you know if there is a time frame for the fix. Perhaps it won't be an issue in a couple months.

**Reply by scadergit** [reply]
I'll bring this up next week to see when NCO can get it scheduled.

**Reply by boofuss60** [reply]
I'm new to this. Maybe obvious but I can't find where that NCO email address is.

Any success in getting the gridpoint error corrections scheduled?
